src/components/Root.tsx

```diff
@@ -33,22 +34,19 @@ function render(props: RootLayoutProps) {
   const displayNavbarLogo = ![''].includes(location)
   const displayNavbarSearch = false && ![''].includes(location)
   const searchShadow = [worksUrl].includes(location)
-  const Main = () => (
-    <Navbar
-      shadow={navbarShadow}
-      displayLogo={displayNavbarLogo}
-      displaySearch={displayNavbarSearch}
-      transparent={navbarTransparent}
-      searchShadow={searchShadow}
-    />
-  )
-  const Test = () => <div>REACT-FEATURE-TOGGLE-TEST</div>
-  const TestFeature = configureFeature(Main, 'foo', Test)
+
+  const FooterFeature = configureFeature(Footer, 'footer', Footer1)
```

Comment:
Having deploy previews, and the footer having no logic at all... isn't a feature toggle for this overkill?

Reply:
I questioned the same thing, but the links aren't yet going anywhere, so it isn't ready in that sense. This was just getting the layout set-up to finish up the landing page design.

Reply:
Got it. 

I'd prefer setting up those links before merging this PR to avoid having the feature toggle, but it looks like they would link to pages that we haven't yet built.

Another approach we could do is: have only one footer and have it hide or show the links (that don't yet link to anything) based on the feature toggle. That would maybe reduce code complexity and avoid naming collisions (Footer and Footer1). Just an idea though.

I'm also OK with merging this one as is if you prefer.

Reply:
Yea I didn't know if we should wait on this until we built those pages or not, I did like the idea of just having the links hidden behind the toggle though. I went ahead an implemented it that way.